CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 
 [Unreleased changes](./UNRELEASED.md)
 
+# 8.0.0
```

Comment:
Missed moving this over from UNRELEASED on the last release, was focusing on the Release Notes in GitHub 🤦